package.json

```diff
@@ -117,6 +117,7 @@
     "@babel/preset-env": "^7.16.11",
     "@pact-foundation/pact": "^9.17.2",
     "@pact-foundation/pact-node": "^10.17.1",
+    "@snyk/protect": "^1.864.x",
```

Comment:
According to https://www.npmjs.com/package/@snyk/protect the latest version is 1.865 — any reason we’re not using that?

Reply:
@alexbishop1 I just did that so we are not changing versions at the same time as the package source to not conflate things I can update it if you prefer.